MobileFilter/sections/specific_web.txt

```diff
@@ -27,6 +27,7 @@ theinventory.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,
 ! ##.ads
 cartoonpornvideos.com,m.economictimes.com,beethovenmaster.com,m.mistrzowie.org,m.ura.news,siirtpress.com,up.ymgt.xyz,google.de,www-bento-de.cdn.ampproject.org,m-economictimes-com.cdn.ampproject.org,m.winfuture.de##.ads
 !
+honto.jp##.momRegBanner 
```

Comment:
Move this rule to Annoyances > self-promo.

Reply:
done